variables.tf

```diff
@@ -182,3 +182,8 @@ variable "executor_docker_auth_config" {
   default     = ""
   description = "If provided, this docker auth config file will be used to authorize image pulls. See [Using private registries](https://docs.sourcegraph.com/admin/deploy_executors#using-private-registries) for how to configure."
 }
+
+variable "randomize_resource_names" {
```

Comment:
shouldn't the default be set to `true`? at least that's how I interpret the comment

Reply:
applies to other places this var is defined as well

Reply:
Part of me feels like the default should be `false` (with the `description` updated to reflect that). I feel most people are not going to really see/understand what this means. So they might get confused/worried that we broke something. 

If/when we do a major version bump, then I feel this could default to `true`.

Reply:
Fair enough! I'll go for `false`